stix_shifter/stix_transmission/src/modules/qradar/arielapiclient.py

```diff
@@ -53,7 +55,7 @@ def __init__(self, connection, configuration):
                                     connection.get('cert', None),
                                     headers,
                                     url_modifier_function,
-                                    connection.get('cert_verify', 'True')
+                                    connection.get('selfSignedCert', True)
```

Comment:
This change would need to happen for all adapters, correct? It's missing from the dummy modules, elastic_ecs, and Splunk.

Reply:
Yes Azam is going to extend it I hope